sim/humanoid_gym/envs/stompy_config.py

```diff
@@ -91,7 +93,8 @@ class init_state(LeggedRobotCfg.init_state):
             default_joint_angles[joint] = default_positions[joint]
 
     class control(LeggedRobotCfg.control):
-        # PD Drive parameters:
+        # PD Drive parameters: THIS
+        # NOTE: these names are outdated, must be manually set in third party legged_robot.py
```

Comment:
Can we update the names here then?

Reply:
issue is, the new joint names no longer have the type of motor they used within their name (x8_shoulder), so its not possible to assign them this way anymore. Since they're all assigned to the same values, for now I amended this to be assigned to their part names (elbow, shoulder, ..)
done

Reply:
nice!